src/icon/busIcon.tsx

```diff
@@ -15,7 +15,7 @@ export const BusIcon = (props: Icon) => (
       <path
         d="M.67 5h20c1 0 1.66 1.66 2.66 1.66h4v4.67a1 1 0 0 1-1 1H4.67L1 11.1a.5.5 0 0 1-.34-.47V5z"
         stroke={props.iconColor}
-        fill={props.iconColor}
+        fill={props.isDisabled ? color.fadedText : props.iconColor}
```

Comment:
Is there a reason not to use [`currentColor`](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/color) in SVG and simply use CSS to change it?

Reply:
I guess none of us were familiar with svgs when we started this, so it's almost never used.
I'll do some cleaning and try to implement it in this icon but for the rest I'm guessing it will need a separate refacto, possibly with improvements to BaseIcon